image.html

```diff
@@ -0,0 +1,2 @@
+<a href="https://www.w3schools.com"><img src="https://www.irishtimes.com/polopoly_fs/1.4513148.1616011727!/image/image.jpg_gen/derivatives/landscape_490/image.jpg" alt="image" width="50" height="50" border=0 align='right'></a> 
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iusto optio nobis exercitationem placeat animi ducimus. Aliquid est quae dolores rerum atque hic tempore quas? Laborum eum explicabo corrupti esse rem!</p>
```

Comment:
You could have used super tags there but this also does the job. I would suggest to always give basic super tags to html file.